plugins/performance-lab/includes/site-health/object-cache/helper.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Helper functions used for Object Cache Support Info.
+ *
+ * @package performance-lab
+ * @since 2.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for Object Cache Info fields.
+ *
+ * @return array Fields.
+ * @since 3.3.0
+ */
+function object_cache_supported_fields() {
+	return array(
+		'extension'        => array(
+			'label' => __( 'Extension', 'performance-lab' ),
+			'value' => wp_get_cache_type(),
+		),
+		'multiple_gets'    => array(
+			'label' => __( 'Multiple gets', 'performance-lab' ),
+			'value' => wp_cache_supports( 'get_multiple' ) ? 'Enabled' : 'Disabled',
+		),
+		'multiple_sets'    => array(
+			'label' => __( 'Multiple sets', 'performance-lab' ),
+			'value' => wp_cache_supports( 'set_multiple' ) ? 'Enabled' : 'Disabled',
+		),
+		'multiple_deletes' => array(
+			'label' => __( 'Multiple deletes', 'performance-lab' ),
+			'value' => wp_cache_supports( 'delete_multiple' ) ? 'Enabled' : 'Disabled',
+		),
+		'flush_group'      => array(
+			'label' => __( 'Flush group', 'performance-lab' ),
+			'value' => wp_cache_supports( 'flush_group' ) ? 'Enabled' : 'Disabled',
+		),
+	);
+}
+
+/**
+ * Attempts to determine which object cache is being used.
+ *
+ * Note that the guesses made by this function are based on the WP_Object_Cache classes
+ * that define the 3rd party object cache extension. Changes to those classes could render
+ * problems with this function's ability to determine which object cache is being used.
+ *
+ * @return string
+ * @since 3.3.0
+ */
+function wp_get_cache_type() {
+	global $_wp_using_ext_object_cache, $wp_object_cache;
```

Comment:
global docblock missing

Reply:
@mukeshpanchal27 I'm not sure what do you mean, can you please clarify?

Reply:
See examples for `@global` at https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/

Reply:
This has been fixed @westonruter @mukeshpanchal27